tests/storage/restore.go

```diff
@@ -808,6 +808,42 @@ var _ = SIGDescribe("[Serial]VirtualMachineRestore Tests", func() {
 				}
 			})
 
+			DescribeTable("should restore a vm with restore size bigger then PVC size", func(restoreToNewVM bool) {
```

Comment:
yeah thats true, also I dont know regarding if this test will run with another provisioner that supports snapshot and doesnt round up it will also fail, currently I think the snapshot tests run only with ceph

Reply:
I wrote a comment about both of what I mentioned so if the test will fail it will help to understand if thats why